pages/learn/glossary.md

```diff
@@ -53,6 +53,19 @@ A property appearing within a [schema](#schema) object.
 
 The [JSON Schema specification](../specification) defines behavior for a large library of keywords which can be used to describe [instances](#instance).
 
+### implementation
+
+An implementation is any piece of software that provides or exposes behavior defined by the JSON Schema specification.
```

Comment:
Taking a stab at addressing @Relequestual's concerns about the wording here...

```suggestion
An implementation is any software tooling or library for working with or evaluating schemas in some way that is not inconsistent with the JSON Schema specification.
```

Reply:
"not inconsistent"?

not just "consistent"?

Reply:
I thought about that, but I thought it was more clear this way despite the double negative. I wanted to convey that an implementation might do more or less than what the spec says as long as as it doesn't contradict it. I felt like "consistent" could be interpreted as referring to the whole spec, which is not necessarily the case for all types of implementations.

But, I'm bad at words, so that might not make any sense.